examples/coinbase/pipeline.yaml

```diff
@@ -0,0 +1,22 @@
+input:
+  label: coinbase
+  websocket:
+    url:  wss://ws-feed.exchange.coinbase.com
+    open_message: '{"type": "subscribe","product_ids": ["ETH-USD","ETH-EUR"],"channels": ["ticker"]}'
+    open_message_type: text
+
+output:
+  http_client:
+    url: http://proton:8123/proton/v1/ingest/streams/tickers
+    verb: POST
+    headers: 
+      Content-Type: application/json
+    batching:
+      count: 10
+      period: 1000ms
+      processors:
```

Comment:
Gimi will enhance the Proton ingest REST API. After that is available, this processor can be removed.

Reply:
this is OK as the default api always works